amti/clis/associate.py

```diff
@@ -0,0 +1,89 @@
+"""Command line interface for associating quals with Workers"""
+
+import logging
+
+import click
+import csv
+
+from amti import actions
+from amti import settings
+from amti import utils
+
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    context_settings={
+        'help_option_names': ['--help', '-h']
+    })
+@click.argument(
+    'ids',
+    type=str,
+    nargs=-1)
+@click.option(
+    '--file', '-f',
+    type=click.Path(exists=True, file_okay=True, dir_okay=False),
+    help="Path to file of WorkerIds.")
+@click.option(
+    '--qual', '-q', 
+    help='QualificationId (or name, if --name flag passed).')
+@click.option(
+    '--value', '-v', 
+    help='Integer value for qual.')
+@click.option(
+    '--name', '-n',
+    is_flag=True,
+    help='Search for qual by name instead of id.')
+@click.option(
+    '--notify',
+    is_flag=True,
+    help='Send notification message about qual.')
+@click.option(
+    '--live', '-l',
+    is_flag=True,
+    help='View the status of HITs from the live MTurk site.')
+def associate_qual(file, ids, qual, name, value, notify, live):
+    """Associate workers with a qualification.
```

Comment:
Would be helpful if you could provide a more complete description of the args as well as what the method returns.